.github/workflows/system-tests-k8s.yml

```diff
@@ -111,7 +111,7 @@ jobs:
         -v /cache:/cache -v /var/sysimage:/var/sysimage -v /var/tmp:/var/tmp
     timeout-minutes: 150
     needs: [system-tests-k8s]
-    if: ${{ github.event_name == 'schedule' }}
+    if: ${{ always() && github.event_name == 'schedule' }}
```

Comment:
I'm a little confused by this logic - why not remove the if statement completely?

Reply:
Then it won't run if `system-tests-k8s` fails which at this point mostly does. I want it to always run after `system-tests-k8s` but only on `schedule` event.

Reply:
Could you add a short comment explaining this? Just so it doesn't get accidentally removed / changed in the future.

Reply:
@marko-k0 won't this always be true? Wouldn't this be more correct?

```
if: ${{ github.event_name == 'schedule'  && always() }}
```

Reply:
Assuming that evaluation is eager and not lazy might have been wrong. I'll search the docs.

Reply:
@nmattia, it works as intended. It's not triggered on `workflow_dispatch`, see [here](https://github.com/dfinity/ic/actions/runs/12030023396).